autorest/models/name_converter.py

```diff
@@ -0,0 +1,186 @@
+import re
```

Comment:
don't put it under "models", let's create a "plugins" folder

Reply:
switched to it's own folder